code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -36,30 +36,35 @@ GLOBAL_LIST_EMPTY(aide_list)
 			new /obj/item/clothing/gloves/gauntlets(src)
 		if(10)
 			new /obj/item/jacobs_ladder(src)
-		if(11)
-			new /obj/item/nullrod/scythe/talking(src)
+		if(11)//select and spawn a random nullrod that a chaplain could choose from
+			var/found = FALSE
+			while(!found)
```

Comment:
There is no possible way for found to be true here making the whole if statement pointless